.github/workflows/plugins.yml

```diff
@@ -88,18 +88,31 @@ jobs:
       - uses: actions/checkout@v2
       - uses: ./.github/actions/testagent/start
       - uses: ./.github/actions/node/setup
-      - run: yarn install --ignore-engines
+      - id: pkg
+        run: |
+          content=`cat ./package.json | tr '\n' ' '`
+          echo "::set-output name=json::$content"
```

Comment:
Just an FYI that the `set-output` command is [deprecated](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/). I've removed it in #3907 so should probably avoid adding it back again here.